src/components/FieldValueDirectAnswer.tsx

```diff
@@ -0,0 +1,122 @@
+import {
+  FieldValueDirectAnswer as FieldValueDirectAnswerType,
+  BuiltInFieldType,
+  Address
+} from '@yext/search-headless-react';
+import { useMemo } from 'react';
+
+/**
+ * Props for FieldValueDirectAnswer.
+ */
+interface FieldValueDirectAnswerProps {
+  /** A field value direct answer result. */
+  result: FieldValueDirectAnswerType,
+  /** Handle onClick event for "View Details" link. */
+  viewDetailsClickHandler?: () => void,
+  /** CSS classes for customizing the component styling. */
+  cssClasses?: FieldValueDirectAnswerCssClasses
+}
+
+/**
+ *  The CSS class interface for FieldValueDirectAnswer.
+ */
+interface FieldValueDirectAnswerCssClasses {
+  header?: string,
+  body?: string,
+  content?: string,
+  answerContainer?: string
+}
+
+/**
+ * Renders a field value direct answer provided by the Search API.
```

Comment:
Could we have an `@internal` here if we don't plan for this component to be public

Reply:
sure!